vfio-ioctls/Cargo.toml

```diff
@@ -28,7 +28,7 @@ kvm-bindings = { version = "0.14.0", optional = true }
 kvm-ioctls = { version = "0.24.0", optional = true }
 thiserror = { workspace = true }
 vfio-bindings = { version = "=0.6.1", path = "../vfio-bindings" }
-vm-memory = { version = "0.16.0", features = ["backend-mmap"] }
+vm-memory = { version = ">= 0.16.0, < 0.18", features = ["backend-mmap"] }
```

Comment:
Why not ping to the exact version, say `0.17.2`?

Reply:
Makes sense - relax version requirements for library crates while enforcing consistent versioning in consumer applications, such as Cloud Hypervisor.

Can you please include a short explanation to the commit message?